osidb/migrations/0081_remove_flaw_is_major_incident.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.19 on 2023-07-03 11:10
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osidb', '0080_alter_flawacknowledgment_affiliation'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='flaw',
+            name='is_major_incident',
+        ),
+    ]
```

Comment:
We try to keep migrations as N-1 compatible as possible, this means that in the scenario in which a deployment fails the database must be kept in a state that is usable both for version N and version N-1.

This change breaks such compatibility because version N-1 will find itself with a missing column in the database.

To amend this, what we do is [deprecate the field](https://github.com/RedHatProductSecurity/osidb/blob/master/docs/developer/DEVELOP.md#deprecate-fields) and then in version N+1 we can actually remove it from the database, because version N and version N-1 will no longer use the field from the database.

This means that is_major_incident can't be a property yet, but you can make a function that will be called when accessing the deprecated field with the `return_instead` argument (grep that in the code for some examples).

Reply:
`return_instead` could just do what your new property does for the time being and the API part can remain intact. The rest can be removed.